repo-scripts/size-analysis/analysis-helper.ts

```diff
@@ -79,7 +79,8 @@ export async function extractDependenciesAndSize(
   map: Map<string, string>
 ): Promise<ExportData> {
   const input = tmp.fileSync().name + '.js';
-  const output = tmp.fileSync().name + '.js';
+  const fulllyResolvedOutput = tmp.fileSync().name + '.js';
+  const minimizedBundleOutput = tmp.fileSync().name + '.js';
```

Comment:
s/fullly/fully

You are using `fullyResolvedOutput` for bundle that resolves all dependencies. Should we change `minimizedBundleOutput` to  something that contrast with it to make the difference apparent for readers, for example, `externalDepsNotResolvedOutput`?